optax/perturbations/_make_pert_test.py

```diff
@@ -45,17 +47,21 @@ def setUp(self):
     rng = np.random.RandomState(0)
 
     self.rng_jax = jax.random.key(0)
```

Comment:
Could we change this test class to not use a persistent random key, this doesn't follow the practice in the rest of the repo. We didn't catch this before.

Could you delete this line and use `parameterized.product` to pass keys to each test, see other comment.